babel.config.js

```diff
@@ -1,3 +1,14 @@
-module.exports = {
-  presets: ['@babel/preset-env', '@babel/preset-typescript']
+const testingConfig = require('./babel.test.config')
+
+module.exports = api => {
+  const isTest = api.env('test');
+
+  if(isTest) return testingConfig;
+
+  return {
+    presets: [
+      '@babel/preset-env',
+      '@babel/preset-typescript'
+    ]
+  };
 };
```

Comment:
Hi @DaniAcu ,  after looking at this, I think we could simplify the these babel config changes to:

```javascript
module.exports = {
  presets: ['@babel/preset-env', '@babel/preset-typescript'],
  env: {
    test: {
      presets: [[
        '@babel/preset-env',
        {
          "targets": {
            "node": "current"
          }
        }
      ], '@babel/preset-typescript'],
    }
  }
};
```

This seems to work and would avoid the additional `babel.test.config.js` file.
What do you think?

Reply:
Solved 😉